src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/RaisingClaimAgainstLitigantInPersonForSpecNotificationHandler.java

```diff
@@ -66,7 +67,8 @@ private CallbackResponse notifyApplicantSolicitorCaseHandedOffline(CallbackParam
     @Override
     public Map<String, String> addProperties(CaseData caseData) {
         return Map.of(
-            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
```

Comment:
We should not be making changes to Spec notifications. Ideally, the Spec workstream should have visibility of this change and make the changes in spec files.

Reply:
Agreed, removed!